content/en/docs/ops/configuration/traffic-management/network-topologies/index.md

```diff
@@ -7,21 +7,35 @@ owner: istio/wg-networking-maintainers
 test: no
 ---
 
-## Configuring network topologies
+## Forwarding external client attributes (IP address, certificate info) to destination workloads
 
 {{< boilerplate experimental >}}
 
-Istio provides the ability to manage settings like [X-Forwarded-For](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_conn_man/headers#x-forwarded-for) (XFF)
-and [X-Forwarded-Client-Cert](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_conn_man/headers#x-forwarded-client-cert)
-(XFCC), which are dependent on how the gateway workloads are deployed. This is currently an in-development feature. For more
-information on `X-Forwarded-For`, see the IETF's [RFC](https://tools.ietf.org/html/rfc7239).
+Many applications require knowing the client IP address and certificate information of the originating request to behave
+properly. Notable cases include logging, tracing audit tools that require the client IP be populated; security tools
+such as Web Application Firewalls (WAF) may need this information to apply rulesets properly. The functionality of
```

Comment:
The linter is picking on the spelling of ruleset.  I see we use it as two words, rule set, in other docs.

Reply:
Fixed